scenario-compiler/src/test/scala/pl/touk/nussknacker/engine/compile/ProcessValidatorSpec.scala

```diff
@@ -1017,7 +1017,7 @@ class ProcessValidatorSpec extends AnyFunSuite with Matchers with Inside with Op
       case Invalid(
             NonEmptyList(
               ExpressionParserCompilationError(
-                "Bad expression type, expected: Record{someInt: Integer, someString: String}, found: Record{someString: String(abc)}",
+                "Bad expression type, expected: Record{someInt: Integer, someString: String}, found: Record{someString: String}",
```

Comment:
We used not proper type for this validation - we can't base on literal values because expressions are replaced with literal placeholders